pkg/build/controller/strategy/util.go

```diff
@@ -404,6 +404,9 @@ func updateConfigsForContainer(c corev1.Container, volumeName string, configDir
 	c.Env = append(c.Env, corev1.EnvVar{Name: "BUILD_REGISTRIES_DIR_PATH", Value: registriesDirPath})
 	c.Env = append(c.Env, corev1.EnvVar{Name: "BUILD_SIGNATURE_POLICY_PATH", Value: signaturePolicyPath})
 	c.Env = append(c.Env, corev1.EnvVar{Name: "BUILD_STORAGE_CONF_PATH", Value: storageConfPath})
+	// Disable blob cache - worker nodes running out of disk
+	// Set BUILD_BLOBCACHE_DIR to empty value per https://github.com/openshift/builder/blob/master/pkg/build/builder/cmd/builder.go#L139
+	c.Env = append(c.Env, corev1.EnvVar{Name: "BUILD_BLOBCACHE_DIR", Value: ""})
```

Comment:
This seems like something the operator should tune based on the size of disks on nodes.  On nodes with small disks the blob cache may not be worth it by default.  On nodes with large disks it's probably always worth it.

Reply:
I can see this being a challenge to do automatically if admins use node selectors to push builds onto specific nodes and/or use multiple machine configs.